### PR TITLE
Improve passport laissez-passer explanation

### PR DESCRIPTION
**Pull request by zGato**
This PR aims to improve and explain in a better way what could happen if "Make the passport a laissez-passer" is enabling either using your own proxy or the ones provided publicly.

### CODE REVIEW

File: src/options/page.html [hunk 1]
```diff
@@ -69,8 +69,28 @@ <h2>Passport</h2>
                   Only enable this option if you are experiencing issues with
                   TTV LOL PRO's passport!
                 </b>
+                <br />
+                </small>
+                <small>
+                <b>
+                  WARNING: Enabling this option while proxying thru another country could enable features that are not enabled on your country (such as Predictions, Prime Subscriptions, Currency Changes and others).
```
Comment (younesaassila) [inline src/options/page.html]: Please change "thru" to "through"

File: src/options/page.html [hunk 2]
```diff
@@ -69,8 +69,28 @@ <h2>Passport</h2>
...
+                <br />
+                <small>
+                <b>
+                  You accept full responsibility for what passes through your proxy or public ones, and be aware that some features such as Predictions could be considered as Gambling in some countries which may be illegal for minors.
+                </b>
+                </small>
+                <br />
+                <small>
+                <b>
+                    If your county doesn't have that feature, it's for a reason. Be aware and search online.
```
Comment (younesaassila) [inline src/options/page.html]: Please fix "county" to "country"

**Review by younesaassila — DISMISSED** [review]
Please fix the two typos found and run the linter by doing `npm run lint:fix`

File: src/options/page.html (hunk 1 above)
Comment (zGato) [reply src/options/page.html]: Fixes commited

**Pull request merged**